src/sql_engine/src/dml/delete.rs

```diff
@@ -38,18 +38,27 @@ impl<P: BackendStorage> DeleteCommand<'_, P> {
     }
 
     pub(crate) fn execute(&mut self) -> SystemResult<QueryResult> {
+        let mut builder = QueryErrorBuilder::new();
         let schema_name = self.name.0[0].to_string();
         let table_name = self.name.0[1].to_string();
         match (self.storage.lock().unwrap()).delete_all_from(&schema_name, &table_name)? {
             Ok(records_number) => Ok(Ok(QueryEvent::RecordsDeleted(records_number))),
-            Err(OperationOnTableError::SchemaDoesNotExist) => Ok(Err(QueryError::schema_does_not_exist(schema_name))),
-            Err(OperationOnTableError::TableDoesNotExist) => Ok(Err(QueryError::table_does_not_exist(
-                schema_name + "." + table_name.as_str(),
-            ))),
+            Err(OperationOnTableError::SchemaDoesNotExist) => {
+                builder.schema_does_not_exist(schema_name);
+                Ok(Err(builder.build()))
```

Comment:
```suggestion
                Ok(Err(QueryErrorBuilder::new().schema_does_not_exist(schema_name).build()))
```

Reply:
Could you please move builder allocation into error branches in other places too